src/galario.cpp

```diff
@@ -677,12 +677,29 @@ inline dcomplex interpolate_core(int const nrow, int const ncol, const dcomplex
     dcomplex const final_add2 = CMPLXADD(term2, term3);
     dcomplex const final_add1 = CMPLXADD(term1, final_add2);
 
-    dcomplex interpolated = CMPLXADD(final_add1, y0);
+    dcomplex phaseInterp = CMPLXADD(final_add1, y0);
```

Comment:
Please change the name of this variable, it's confusing since it's a complex number, not only the phase.
I would call it, e.g. `interpolated_ReIm` so that `angleInterp` below can be properly renamed `phaseInterp`.